### PR TITLE
Feature/words highlighting ui draft

### PR DESCRIPTION
**Pull request by alxmglk**

### CODE REVIEW

**Comment by mikhailmikhaylov** [comment]
Should I review this already?

Are we going to merge this before PR#1?

**Comment by mikhailmikhaylov** [comment]
Can we select multiple words? Can we select all unknown words?

File: src/main/webapp/js/highlighting.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+			
+			highlightedText = highlightedText.slice(0, endIndex) + wrapper.endTag + highlightedText.slice(endIndex);
+			highlightedText = highlightedText.slice(0, startIndex) + wrapper.startTag + highlightedText.slice(startIndex);
+		});
+		
+		$textWithHighlighting.html(highlightedText);
+	};
+	
+	var loadDummyText = function () {
+		$.get('data/highlighting/text.txt', function (dummyText) {
```
Comment (mikhailmikhaylov) [inline src/main/webapp/js/highlighting.js:84]: Please provide some info about proper CORS setup

Comment (mikhailmikhaylov) [inline src/main/webapp/js/highlighting.js:84]: nvm, I've found a way to run it

**Comment by mikhailmikhaylov** [comment]
LGTM

**Comment by alxmglk** [comment]
yep, it's possible to select any number of unknown words in the list

**Comment by alxmglk** [comment]
I think it's ok to merge this specific branch first

**Pull request merged**